experimental/Schemes/BlowupMorphism.jl

```diff
@@ -0,0 +1,255 @@
+export BlowupMorphism
+export center, projection, exceptional_divisor
+
+mutable struct BlowupMorphism{
+                              CodomainType<:AbsCoveredScheme
+                             }
+  projective_bundle::CoveredProjectiveScheme 
+  codomain::CodomainType   # i.A. ein CoveredScheme
+  center::IdealSheaf      # on codomain
+  projection::AbsCoveredSchemeMorphism
+  domain::AbsCoveredScheme # i.A. ein CoveredScheme
+  exceptional_divisor::EffectiveCartierDivisor
+
+  function BlowupMorphism(
+      IP::CoveredProjectiveScheme,
+      I::IdealSheaf
+    )
+    X = base_scheme(IP)
+    X === scheme(I) || error("ideal sheaf not compatible with blown up variety")
+    return new{typeof(X)}(IP, X, I)
+  end
+end
+
+function domain(p::BlowupMorphism)
+  if !isdefined(p, :domain)
+    p.domain = covered_scheme(p.projective_bundle)
+  end
+  return p.domain
+end
+
+codomain(p::BlowupMorphism) = p.codomain
+center(p::BlowupMorphism) = p.center
+
+function projection(p::BlowupMorphism)
+  if !isdefined(p, :projection)
+    p.projection = covered_projection_to_base(p.projective_bundle)
+  end
+  return p.projection
+end
+
+# TODO: Find better name!
+covered_projective_scheme(p::BlowupMorphism) = p.projective_bundle
+
+@Markdown.doc """
+    exceptional_divisor(p::BlowupMorphism)
```

Comment:
p.exceptional_divisor seems to be set upon creation of p, while the code here is deprecated (error-statement in line 58). This needs to be cleaned up.
Stupid question: How are multiplicities of excpetional divisors in total transforms of subschemes handled?

Reply:
In which sense? It's an `EffectiveCartierDivisor`. If you convert this to a `WeilDivisor`, it will have multiplicities in general. But I don't know how to extract the multiplicity from an `EffectiveCartierDivisor` directly. 

What precisely did you have in mind with your question?

Reply:
Let's talk about this in person later.